trade/assets/js/service.js

```diff
@@ -0,0 +1,11 @@
+
+
+export function fetchCurrencies() {
```

Comment:
use const var for baseUrl = "https://marketdata.tradermade.com/api/v1/" and apiKay = "i4ydISb-yX--K-I-AMjV"

Reply:
done